MIPs/MIP-003/MIP-003.md

```diff
@@ -165,7 +164,6 @@ GET https://example-url.com/status?job_id=job_456abc
 {
   "job_id": "job_456abc",
   "status": "awaiting_input",
-  "paybytime": 1721480200,
```

Comment:
I think it makes sense to have it? Potentially add all payment details too?

Otherwise could this break some existing flows? (for masumi it wont) @PatrickTobler